runelite-client/src/test/java/net/runelite/client/plugins/chatcommands/ChatCommandsPluginTest.java

```diff
@@ -1152,6 +1152,24 @@ public void testCounters()
 			"Fastest Room time - (Team size: (5 player): <col=ffffff>18:45</col>",
 			"Fastest Overall time - (Team size: 5 player): <col=ffffff>22:01</col>",
 			"",
+			"Tombs of Amascut - Expert",

```

Comment:
Why is this missing non-expert toa? Did you pull this from an actual dump of the in-game text or make it up?